setup-scripts/skynet-nginx.conf

```diff
@@ -1,3 +1,7 @@
+limit_req_zone $binary_remote_addr zone=stats:10m rate=10r/m;
+limit_conn_zone $binary_remote_addr zone=uploads:10m;
+limit_conn_zone $binary_remote_addr zone=downloads:10m;
```

Comment:
PP: I think it makes more sense to name the zones by the aggregator. 
So I would've probably defined a zone `byIPAddress`
And then in every location define the rate through the `limit_conn` directive.